src/vg/civcraft/mc/citadel/Utility.java

```diff
@@ -418,10 +418,22 @@ public static boolean reinforcementDamaged(Player player, Reinforcement reinforc
         reinforcement.setDurability(durability);
         boolean cancelled = durability > 0;
         if (durability <= 0) {
-            cancelled = reinforcementBroken(null, reinforcement);
+			if (CitadelConfigManager.shouldLogHostileBreaks()) {
```

Comment:
That method returning true or false doesn't indicate that the reinforcement was broken or not; it simply controls if after the reinf is deleted, does the original block drop or not.

I agree this isn't perfectly ideal but it's close to it. Eventually I'd like to refactor the lot of this and make everything a good deal more explicit.